content/en/docs/use-cases/aws-codecommit.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Using Flux on AWS With CodeCommit
+linkTitle: AWS CodeCommit
+description: "How to boostrap Flux on AWS EKS with CodeCommit Git repositories."
+weight: 10
+---
+
+## Cluster Creation
+
+The following creates an EKS cluster with some minimal configuration that will work well with Flux:
+
+```sh
+eksctl create cluster
+```
+
+{{% alert color="info" %}}
+When working with `eksctl` you can provide a `ClusterConfig`, using `-f`, to setup how you want your cluster to be
```

Comment:
Please replace this with something like "For more details on how to create EKS cluster with `eksctl` please see https://eksctl.io"